go/parquet/pqarrow/encode_arrow_test.go

```diff
@@ -427,6 +427,44 @@ func TestWriteEmptyLists(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestWriteAllNullsWithDeltaEncoding(t *testing.T) {
+	sc := arrow.NewSchema([]arrow.Field{
+		{Name: "f1", Type: arrow.PrimitiveTypes.Int64, Nullable: true},
+		{Name: "f2", Type: arrow.ListOf(arrow.FixedWidthTypes.Date32)},
```

Comment:
Hmm why List is tested here?

Reply:
Strictly speaking this is not necessary，I just copy the previous test and want to add some simple tests on nested formats.

Reply:
It's good to ensure we're testing nulls for multiple types